pyflakes/messages.py

```diff
@@ -116,6 +116,22 @@ def __init__(self, filename, loc, name):
         self.message_args = (name,)
 
 
+class DuplicateDictionaryKey(Message):
+    message = 'dictionary key %s duplicated'
```

Comment:
Sure, I'll be looking at all comments and updating the PR/comments (hopefully) appropriately this evening.